addons/medical_treatment/functions/fnc_loadUnit.sqf

```diff
@@ -33,8 +34,20 @@ if (_patient call EFUNC(medical_status,isBeingDragged)) then {
 
 private _vehicle = [_medic, _patient, _vehicle] call EFUNC(common,loadPerson);
 
-if (!isNull _vehicle) then {
-    private _patientName = [_patient, false, true] call EFUNC(common,getName);
-    private _vehicleName = getText (configFile >> "CfgVehicles" >> typeOf _vehicle >> "displayName");
-    [[LSTRING(LoadedInto), _patientName, _vehicleName], 3] call EFUNC(common,displayTextStructured);
+if (isNull _vehicle) then {
```

Comment:
If you want to avoid the if / else combination you can write it like `if (isNull _vehicle) exitWith { TRACE_1("no vehicle found",_vehicle);};` which may be cleaner at the end.